### DISCUSSION TITLE
Password authentication

### DISCUSSION THREAD

**Discussion by Anifredgithub** [community/community#49504] [New to GitHub]
Authentication: i changed my password and this has been affecting ALX project. during my programming I'm asked to input my password and when i do, i get the message that my password cannot be authenticated in quote " Support for password authentication was removed on August 13, 2021. remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication. fatal: Authentication failed for 'https://github.com/anifredgithub/alx-zero_day.git/'"

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Comment by Anifredgithub** [comment]
Yes, I've also done that and it still does not authenticate PAT.
